app/src/main/java/com/tapptitude/pageindicator/sample/MainActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.tapptitude.pageindicator.sample
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.recyclerview.widget.RecyclerView.HORIZONTAL
+import androidx.recyclerview.widget.RecyclerView.VERTICAL
+import com.tapptitude.pageindicator.sample.databinding.ActivityMainBinding
+import com.tapptitude.pageindicator.sample.util.viewBinding
+
+class MainActivity : AppCompatActivity() {
+
+    private val binding by viewBinding(ActivityMainBinding::inflate)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        binding.pageIndicator.initPageIndicator(7, indicatorSpacing = 12)
```

Comment:
as the sample might also progress in the future I suggest we extract this into constants